document/CoreFormat.mdk

```diff
@@ -114,16 +114,21 @@ An `sarifLog` object specifies the version of the file format and contains the o
 
 ### `version` property { #sarifLog-version }
 
-An `sarifLog` object shall contain a property named `version` whose value is the string `"0.4"`.
+A `sarifLog` object shall contain a property named `version` whose value is the string `"1.0.0-beta.1"`.
 
 Although the order in which the name/value pairs appear in a JSON object value is not semantically significant,
-the `version` property should appear first.
-
+the `version` property should appear first. 
```

Comment:
Restore the deleted blank line. (Affects only the readability of the markdown, not the format of the HTML.)